plugin/mutate/source/dextool/plugin/mutate/backend/report/html/page_stats.d

```diff
@@ -61,13 +61,15 @@ void overallStat(const MutationStat s, Element base) {
             tuple("Total", s.total),
             tuple("Killed by compiler", cast(long) s.killedByCompiler),
             tuple("Skipped", s.skipped), tuple("Equivalent", s.equivalent),
-            tuple("Worklist", cast(long) s.worklist),
         ]) {
         tbl.appendRow(d[0], d[1]);
     }
-
-    base.addChild("p").appendHtml(
-            "<i>worklist</i> is the number of mutants that are in the queue to be tested/retested.");
+    {
+        auto foo = tbl.appendRow;
+        string infoText = "Worklist is the number of mutants that are in the same queue to be tested/retested";
```

Comment:
because the string is only used at one place, line 70, the variable is unnecessary. move the string to the one place it is used.